cmd/restic/cmd_backup.go

```diff
@@ -141,6 +142,7 @@ func init() {
 	f.BoolVar(&backupOptions.IgnoreInode, "ignore-inode", false, "ignore inode number changes when checking for modified files")
 	f.BoolVar(&backupOptions.IgnoreCtime, "ignore-ctime", false, "ignore ctime changes when checking for modified files")
 	f.BoolVarP(&backupOptions.DryRun, "dry-run", "n", false, "do not upload or write any data, just show what would be done")
+	f.BoolVar(&backupOptions.DisableScanner, "disable-scanner", false, "do not run scanner to estimate size of backup")
```

Comment:
After discussing with @rawtaz, we think that `--no-scan` would be more in line with the other options that disable something (e.g. `--no-cache`, `--no-lock`). Please rename that option and the attribute in the `BackupOptions` struct.